cpu.c

```diff
@@ -345,7 +347,7 @@ static void __gb_step_cpu(struct gb_s *gb)
         break;
 
     case 0x10: /* STOP */
-        // gb->gb_halt = 1;
+        gb->gb_halt = 1;
```

Comment:
You have to explain the change.

Reply:
![image](https://user-images.githubusercontent.com/33973536/103778213-e692c300-506c-11eb-9237-fbb8cdbb9514.png)

since gbit function will compare `STOP` instruction, i need to change this instruction to avoid mismatch

Reply:
Don't put the picture which only consists of ASCII characters.

Reply:
Split the change of halt in a separate commit with explanation.

Reply:
Send another pull request to correct the emulation of `STOP` instruction.